packages/agents/sequencer/src/lib/operations/auctions.ts

```diff
@@ -194,6 +259,12 @@ export const executeAuctions = async (_requestContext: RequestContext) => {
               origin,
               destination,
             });
+
+            // Update router liquidity record to reflect spending.
+            routerLiquidity = routerLiquidity.sub(amount);
+            await cache.routers.setLiquidity(router, destination, asset, routerLiquidity);
```

Comment:
I see liquidity is being removed here, but it seems like it won't get re-added on its own, it will expire itself then we rely on the value expiring and then getting re-queried? I guess that works I just don't fully see why we need to cache the liquidity at all.

Reply:
you're right that it could probs just be set in memory in some local variable/mapping. I just thought that using the cache was a better model for storage, even if it's truly just temporary.